AM603.xml

```diff
@@ -0,0 +1,74 @@
+<rom base="NISSAN_BASE">
+	<romid>
+		<xmlid>AM603</xmlid>
+		<hwid>3MFED18N8</hwid>
+		<internalidaddress>6c31</internalidaddress>
+		<internalidstring>AM603</internalidstring>
+		<ecuid>AM603</ecuid>
+		<year>03</year>
+		<market>USDM</market>
+		<make>Infiniti</make>
+		<model>G35</model>
+		<submodel>Base</submodel>
+		<transmission>AT</transmission>
+		<memmodel>SH7055</memmodel>
+		<flashmethod>nisprog</flashmethod>
+		<filesize>512kb</filesize>
+	</romid>
+
+	<!-- very similar to 5Y702, similar to 6Y300 -->
+	<!-- initial def : fenugrec 2018 -->
```

Comment:
please don't just copy-paste stuff, like this comment, is it still true that this is similar to 5y702 ? For sure it wasn't done by me.
Also, if it is identical to AM604, then you should be using rom base=AM604 instead of copying the whole thing.